frontend/src/scenes/data-management/event-properties/EventPropertyDefinitionsTable.tsx

```diff
@@ -0,0 +1,113 @@
+import './EventPropertyDefinitionsTable.scss'
+import React from 'react'
+import { useActions, useValues } from 'kea'
+import { LemonTable, LemonTableColumn, LemonTableColumns } from 'lib/components/LemonTable'
+import { PropertyDefinition } from '~/types'
+import { SceneExport } from 'scenes/sceneTypes'
+import { ObjectTags } from 'lib/components/ObjectTags/ObjectTags'
+import { organizationLogic } from 'scenes/organizationLogic'
+import { PropertyDefinitionHeader } from 'scenes/data-management/events/DefinitionHeader'
+import { humanFriendlyNumber } from 'lib/utils'
+import {
+    EVENT_PROPERTY_DEFINITIONS_PER_PAGE,
+    eventPropertyDefinitionsTableLogic,
+} from 'scenes/data-management/event-properties/eventPropertyDefinitionsTableLogic'
+
+export const scene: SceneExport = {
+    component: EventPropertyDefinitionsTable,
+    logic: eventPropertyDefinitionsTableLogic,
+    paramsToProps: () => ({ syncWithUrl: true }),
+}
+
+export function EventPropertyDefinitionsTable(): JSX.Element {
+    const { eventPropertyDefinitions, eventPropertyDefinitionsLoading, openedDefinitionId } = useValues(
+        eventPropertyDefinitionsTableLogic
+    )
+    const { loadEventPropertyDefinitions, setLocalEventPropertyDefinition } = useActions(
+        eventPropertyDefinitionsTableLogic
+    )
+    const { hasDashboardCollaboration, hasIngestionTaxonomy } = useValues(organizationLogic)
+
+    const columns: LemonTableColumns<PropertyDefinition> = [
+        {
+            title: 'Name',
+            key: 'name',
+            className: 'definition-column-name',
+            render: function Render(_, definition: PropertyDefinition) {
+                return (
+                    <PropertyDefinitionHeader
```

Comment:
Usually icons are in their own columns (e.g. in Trends), though I see how this can be more difficult here
<img width="146" alt="Screen Shot 2022-03-11 at 15 11 57" src="https://user-images.githubusercontent.com/4550621/157884708-c2476cfc-2418-4dfc-9338-caeff19f5f9e.png">